tests/ecp5/latches.ys

```diff
@@ -0,0 +1,16 @@
+read_verilog latches.v
+design -save read
+
+proc
+async2sync # converts latches to a 'sync' variant clocked by a 'super'-clock
```

Comment:
Is it still appropriate to use `async2sync` for latches?

Reply:
I tried to use -multiclock option and don't use async2sync for this test but equiv_opt also reports an error about unproven cells.

Reply:
Using `async2sync` will create `$_FF_` cells (which aren't being counted because the resource count is on the synthesis without `async2sync`) so this isn't testing the correctness of latch synthesis. I expect the problem is that `equiv_induct` can't handle combinational loops.

Reply:
Can we make some changes in equiv_induct for combinational loops?

Reply:
Hmm... am struggling to see how we can verify equivalence here. Since latches are blown into soft logic (unlike in Xilinx where it is captured inside an `LD*` cell) meaning that the presence of any `$dlatch` is lost...